tasks.py

```diff
@@ -32,18 +32,32 @@ def is_truthy(arg):
     return bool(strtobool(arg))
 
 
-PYTHON_VER = os.getenv("PYTHON_VER", "3.7")
-
+default_python_ver = "3.7"
+
+# Use pyinvoke configuration for default values, see http://docs.pyinvoke.org/en/stable/concepts/configuration.html
+# Variables may be overwritten in invoke.yml or by the environment variables INVOKE_NAUTOBOT_xxx
+namespace = Collection("nautobot")
+namespace.configure(
+    {
+        "nautobot": {
+            "python_ver": default_python_ver,
+            "invoke_local": False,
```

Comment:
Now that this maps to an environment variable perhaps just name this `local` so that the var maps to `INVOKE_NAUTOBOT_LOCAL` which IMO reads better and doesn't have "invoke" in it"